common/src/main/java/uk/gov/gchq/palisade/rule/PredicateRule.java

```diff
@@ -28,7 +28,7 @@
  */
 public interface PredicateRule<T> extends Rule<T> {
     /**
-     * Applies the rule logic to test whether a record should be redacted based on the user and justification.
+     * Applies the rule logic to test whether a record should be redacted based on the user and purpose.
```

Comment:
This should actually refer to the context. looks like legacy comments that were not changed when we expanded to context rather than just having a justification.